claat/fetch/fetch.go

```diff
@@ -171,17 +172,19 @@ func (f *Fetcher) SlurpImages(src, dir string, steps []*types.Step) (map[string]
 	}
 
 	imap := make(map[string]string, count)
-	var err error
+	errBuilder := strings.Builder{}
 	for i := 0; i < count; i++ {
 		r := <-ch
 		imap[r.file] = r.url
-		if r.err != nil && err == nil {
-			// record first error
-			err = fmt.Errorf("%s => %s: %v", r.url, r.file, r.err)
+		if r.err != nil {
+			errBuilder.WriteString(fmt.Sprintf("%s => %s: %v", r.url, r.file, r.err))
```

Comment:
I think these append them [without newlines](https://golang.org/src/strings/builder.go). Can you add newlines or some separator?